shared/video-sdk/develop/product-workflow/project-setup/react-native.mdx

```diff
@@ -1,3 +1,18 @@
 <PlatformWrapper platform="react-native">
 
+If your target platform is iOS:
+
+1. Open the `.xcworkspace` file under the ios folder with Xcode.
+1. Create a Broadcast Upload extension and make sure "Include UI Extension" is unchecked.
+    - File > New > Target > Broadcast upload extension
+1. Set the `NSExtensionPrincipalClass` property of the `Info.plist` file inside the target to `AgoraReplayKitHandler`.
+1. To include `Agora RTC` dependency to the newly created Broadcast Upload extension target, add the following lines at the end of the iOS Podfile:
+
+```swift
+target 'ScreenShareExtension' do
+pod 'AgoraIrisRTC_iOS', '4.0.0-rc.1'
```

Comment:
That version number no longer exists. Need to put in a metadata thing.

Reply:
Resolved